cmd/manager/main.go

```diff
@@ -59,6 +62,26 @@ func main() {
 
 	printVersion()
 
+	// Check if we need to watch a specific namespace
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		log.Error(err, "Failed to get watch namespace")
+		os.Exit(1)
+	}
+
+	if strings.Contains(namespace, ",") {
+		namespace = ""
+		log.Info(`This operator cannot handle multiple
+			  namespaces. The operator will watch for changes
+		          across all namespaces`)
+	}
+
+	if namespace == "" {
+		namespace = operatortypes.OperatorNamespace
+		log.Info(`This operator must watch resources in a single
+                          namespace. Setting default namespace value`)
```

Comment:
it might be helpful to log the the namespace value itself

Reply:
also, does this mean that we won't reconcile changes in pod controller since it watches resources in nsx-system ns?

Reply:
hmm, I think I need to make it work since it will expect now that the pod and daemonset are in the same namespace as the operator